src/Kernel.EFSupport/Helpers/ConnectionStringExtension.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using LT.DigitalOffice.Kernel.Helpers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Serilog;
+
+namespace LT.DigitalOffice.Kernel.EFSupport.Helpers
+{
+  public static class ConnectionStringExtension
+  {
+    public static string AddConnectionString(this IServiceCollection services, IConfiguration _configuration)
```

Comment:
rename to GetConnectionString
what for   IServiceCollection here?

Reply:
"GetConnectionString" - OK;
"this IServiceCollection services" for using this like extension in ConfigureServices (services.GetConnectionString);
don't we need this?

Reply:
IServiceCollection deleted